src/main/java/AllCommands/AddCommand.java

```diff
@@ -0,0 +1,53 @@
+package AllCommands;
+
+import BasisSupport.Status;
+import BasisSupport.transformString;
+import Support.*;
```

Comment:
While I understand that you require all objects within the Support package, consider adhering to the module's coding standard (state each individual import). This is intended to help improve code readability and clarity.